src/components/Observations/ObsCardStats.js

```diff
@@ -1,74 +1,72 @@
 // @flow
 
+import classnames from "classnames";
 import checkCamelAndSnakeCase from "components/ObsDetails/helpers/checkCamelAndSnakeCase";
-import { ActivityCount, QualityGradeStatus } from "components/SharedComponents";
-import { View } from "components/styledComponents";
+import { ActivityCount, Body3, QualityGradeStatus } from "components/SharedComponents";
+import { Image, View } from "components/styledComponents";
 import type { Node } from "react";
 import React from "react";
 import { useTranslation } from "react-i18next";
 import { useTheme } from "react-native-paper";
 
 type Props = {
   item: Object,
```

Comment:
I know this is a bit out of scope, but let's change this prop to `observation` since `item` is always an observation here.